src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_validators.py

```diff
@@ -408,6 +392,64 @@ def process_blob_copy_batch_namespace(namespace):
     if namespace.prefix is None and not namespace.recursive:
         raise ValueError('incorrect usage: --recursive | --pattern PATTERN')
 
+
+def process_file_upload_batch_parameters(namespace):
+    """Process the parameters of storage file batch upload command"""
+
+    # 1. quick check
+    if not os.path.exists(namespace.source):
+        raise ValueError('incorrect usage: source {} does not exist'.format(namespace.source))
+
+    if not os.path.isdir(namespace.source):
+        raise ValueError('incorrect usage: source must be a directory')
```

Comment:
Also inconsistent: "incorrect usage: --source must be a directory" (or change the message in the blob method works too)